codeclimate-govet.go

```diff
@@ -1,30 +1,45 @@
 package main
 
-import "github.com/codeclimate/cc-engine-go/engine"
-import "strings"
-import "os"
-import "os/exec"
-import "strconv"
+import (
+	"github.com/codeclimate/cc-engine-go/engine"
+	"strings"
+	"os"
+	"os/exec"
+	"strconv"
+	"fmt"
+)
 
 func main() {
 	rootPath := "/code/"
 
 	config, err := engine.LoadConfig()
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error loading config: %v\n", err)
 		os.Exit(1)
 	}
 
 	analysisFiles, err := engine.GoFileWalk(rootPath, engine.IncludePaths(rootPath, config))
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error initializing: %v\n", err)
 		os.Exit(1)
 	}
 
 	for _, path := range analysisFiles {
 		cmd := exec.Command("go", "tool", "vet", path)
 
 		out, err := cmd.CombinedOutput()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error analyzing path: %v\n", path)
+			fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+
+			if out != nil {
+				s := string(out[:])
+				fmt.Fprintf(os.Stderr, "Gofmt output: %v\n", s)
```

Comment:
so this is the case where the command fails, but we still got some output, and maybe that's of-interest?

should this say "govet output"?